src/payment/strategies/mollie/mollie-initialize-options.ts

```diff
@@ -1,7 +1,65 @@
+/**
+ * A set of options that are required to initialize the Mollie payment method.
+ *
+ * Once Mollie payment is initialized, credit card form fields are provided by the
+ * payment provider as IFrames, these will be inserted into the current page. These
+ * options provide a location and styling for each of the form fields.
+ *
+ * ```js
+ * service.initializePayment({
+ *      methodId: 'mollie',
+ *      mollie: {
+ *          containerId: 'container',
+ *          cardNumberId: '',
+ *          cardHolderId: '',
+ *          cardCvcId: '',
+ *          cardExpiryId: '',
+ *          styles : {
+ *              base: {
+ *                  color: '#fff'
+ *              }
+ *          }
+ *      }
+ * });
+ * ```
+ *
+ * ContainerId is use in Mollie for determined either its showing or not the
```

Comment:
This paragraphp is not very clear to me. If Mollie has vaulted instruments, does that mean containerId is mandatory? Also, this explanation could go in the docblock below, replacing the
```
The location where Mollie Components are inserted
```

Reply:
when mollie has vaulted instruments we shouldnt create MollieComponents because there is no where to mount them, but when you select add another card it should create and mount MollieComponents 

thats why i added the containerId so i can know when its hidden or shown